packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -502,14 +502,6 @@ class TimelineProtocol {
       timelineController.addFrame(frame);
       pendingFrames.remove(frame.id);
       frame.addedToTimeline = true;
```

Comment:
add a TODO to add back prefetching but only when the flutter app is idle.

Reply:
done.